org.scala-ide.sdt.core.tests/src/org/scalaide/core/classpath/ClasspathTests.scala

```diff
@@ -240,7 +297,8 @@ class ClasspathTests {
       expectedWarnings = 0, expectedErrors = 2)
   }
 
-  /** Major binary-incompatible library on the classpath, with and without version suffix of their own.
+  /**
```

Comment:
We should converge on one style for comments. I am fine with both, so we can go for this one if you prefer it.